web/modules/custom/portland_migrations/migrations/auditor_blog_redirects.yml

```diff
@@ -0,0 +1,34 @@
+id: audit_blog_redirects
+label: 'Migrate Auditor Blog legacy paths into Redirects table'
+langcode: en
+migration_group: auditor
```

Comment:
The migration_group is used to link together all migrations that are related to each other and should be run together. So for this migration, it should probably be set to 'auditor_blog'.